examples/human_in_the_loop.py

```diff
@@ -82,13 +92,7 @@ def act(self, obs):
         return self.action
 
 
-def main(
-    scenarios,
-    sim_name,
-    headless,
-    num_episodes,
-    seed,
-):
+def main(scenarios, sim_name, headless, num_episodes, seed):
```

Comment:
What version of black did you use?

Reply:
`black --version` -> `black, version 19.3b0`